Formula/semgrep.rb

```diff
@@ -137,6 +137,14 @@ def install
 
       system "opam", "exec", "--", "make", "setup"
 
+      # Install spacegrep
+      cd "spacegrep" do
```

Comment:
Please add a test for this.

Reply:
When semgrep initializes it verifies that semgrep-core and spacegrep are on path. https://github.com/returntocorp/semgrep/blob/684302680fa07c6f7a884750c8efaad0df777587/semgrep/semgrep/util.py#L148 so existing tests will fail if this is not installed correctly